addons/binding/org.openhab.binding.kostalinverter/ESH-INF/i18n/kostalinverter_de.properties

```diff
@@ -0,0 +1,32 @@
+# binding
+binding.kostalinverter.name = Kostal Inverter Binding
+binding.kostalinverter.description = Dieses Binding liest die Kostal Inverter Daten �ber die Weboberfl�che aus.
+
+# thing types
+thing-type.kostalinverter.kostalinverter.description = Kostal Inverter Liefert Daten wie z.B. Energie und Leistung.
+
+# thing types config
+thing-type.config.kostalinverter.kostalinverter.url.label = URL
+thing-type.config.kostalinverter.kostalinverter.url.description = URL der Kostal Inverter Weboberfl�che.
+
+thing-type.config.kostalinverter.kostalinverter.userName.label = Benutzer
+thing-type.config.kostalinverter.kostalinverter.userName.description = Benutzer zur Authentifizierung an der Kostal Inverter Weboberfl�che.
+
+thing-type.config.kostalinverter.kostalinverter.password.label = Passwort
+thing-type.config.kostalinverter.kostalinverter.password.description = Passwort zur Authentifizierung an der Kostal Inverter Weboberfl�che.
+
+thing-type.config.kostalinverter.kostalinverter.refreshInterval.label = Abfrageintervall
+thing-type.config.kostalinverter.kostalinverter.refreshInterval.description = Intervall zur Abfrage des Daten (in s).
+
+# channel types
+channel-type.kostalinverter.dayEnergy.label = Gesamtverbrauch (24h)
```

Comment:
It is called "Tagesenergie" in the web interface.
But I agree "Verbrauch" is better.
Maybe "Tagesverbrauch"?

And it would rather be "this day so far" Than "24h".

see: https://openhabdoc.readthedocs.io/de/latest/Kostal_inverter_HTTP/